code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,38 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
+	blend_mode = BLEND_MULTIPLY
+
+/atom/movable/screen/fullscreen/high/New(...)
+
+	. = ..()
+
+	alpha = 0
+
+	animate(
+		src,
+		alpha = 255,
+		time = 10 SECONDS
+	)
+
+	animate(
+		src,
+		color = color_matrix_rotate_hue(0),
+		loop = -1,
+		time = 2 SECONDS
+	)
+
+	var/step_precision = 18 //Larger is more precise rotations.
+	for(var/current_step=1,current_step<=step_precision-1,current_step++) //We do the -1 here because 360 == 0 when it comes to angles.
```

Comment:
```suggestion
	for(var/current_step=1 , current_step <= step_precision - 1, current_step++) //We do the -1 here because 360 == 0 when it comes to angles.
```
But also, there doesn't seem to be a reason why this is c-styled instead of dm-styled
```suggestion
	for(var/current_step in 1 to step_precision - 1) //We do the -1 here because 360 == 0 when it comes to angles.
```